hera_qm/xrfi.py

```diff
@@ -410,13 +482,13 @@ def flag(uvf_m, nsig_p=6., nsig_f=None, nsig_t=None, avg_method='quadmean'):
 
     # Pixel flagging
     if nsig_p is not None:
-        uvf_f.flag_array[uvf_m.metric_array > nsig_p] = True
+        uvf_f.flag_array[uvf_m.metric_array >= nsig_p] = True
```

Comment:
The "or equal" allows us to use 1 as a threshold when flagging on fraction of antenna/baselines that flagged a specific pixel.